shared/cfg/guest-os/Windows.cfg

```diff
@@ -276,7 +276,7 @@
         analyze_cmd = E:\autoit3.exe E:\dump_control.au3
         sub_type = nmi_bsod_catch
         rename_error_image = yes
-    balloon_check:
+    balloon_check, balloon_hotplug:
         free_mem_cmd = wmic os get FreePhysicalMemory
```

Comment:
There is no problem for the cmd, as there is a ratio to handle the free mem difference for above 2 cases.